dev_fixtures_do_not_merge/index.mjs

```diff
@@ -0,0 +1,3 @@
+import symbol from 'yyy/lib.mjs';
```

Comment:
You can put these fixtures in https://github.com/nodejs/node/tree/main/test/fixtures/es-module-loaders if you’d like, or potentially use ones already there if any can work for your purposes. We’ll need some tests before adding this feature, and those tests will need fixtures (either these new ones or existing ones).

Reply:
Turned them into a proper test